server/user/user.model.js

```diff
@@ -1,22 +1,27 @@
 'use strict';
 
-const { auth: config = {} } = require('../config');
-const { Model, Sequelize, Op, UniqueConstraintError } = require('sequelize');
-const { role } = require('../../common/config');
-const { sql } = require('../common/database/helpers');
+const { Model, Op, Sequelize, UniqueConstraintError } = require('sequelize');
 const Audience = require('../common/auth/audience');
 const bcrypt = require('bcrypt');
 const castArray = require('lodash/castArray');
+const config = require('../config');
 const find = require('lodash/find');
+const IntervalCache = require('../common/util/interval-cache');
 const jwt = require('jsonwebtoken');
 const logger = require('../common/logger')();
 const mail = require('../common/mail');
 const map = require('lodash/map');
 const pick = require('lodash/pick');
 const Promise = require('bluebird');
-const Role = require('../../common/config/role');
+const { role } = require('../../common/config');
```

Comment:
Enum likes should be PascalCase

Reply:
Updated `IntervalCache.EVENTS` to `Events`, too.